src/cmd/CMakeLists.txt

```diff
@@ -1,3 +1,86 @@
+# Collect source files into the "sources" variable and unit test files into the
+# "gtest_sources" variable.
+gz_get_libsources_and_unittests(sources gtest_sources)
+
+add_library(gz STATIC gz.cc)
+target_link_libraries(gz
+  PUBLIC
+    ${PROJECT_LIBRARY_TARGET_NAME}
+)
+
+set(model_executable gz-sim-model)
+add_executable(${model_executable} model_main.cc)
+target_link_libraries(${model_executable}
+  gz
+  gz-utils${GZ_UTILS_VER}::cli
+  ${PROJECT_LIBRARY_TARGET_NAME}
+)
+
+install(
+  TARGETS ${model_executable}
+  DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/gz/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}/)
+
+# Build the unit tests.
+set(test_sources)
+
+# gz_TEST and ModelCommandAPI_TEST are not supported with multi config
+# CMake generators, see also cmd/CMakeLists.txt
+get_property(GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+if(NOT GENERATOR_IS_MULTI_CONFIG)
+  list(APPEND test_sources
+    ../gz_TEST.cc
+    ../ModelCommandAPI_TEST.cc
+  )
+endif()
+
+gz_build_tests(TYPE UNIT
```

Comment:
We need to make sure we build the `ModelCommandAPI_TEST` if a valid display is found. Previously, it was added to the `tests_needing_display` variables